tests/perf-test/ReportScreen.perf-test.tsx

```diff
@@ -173,83 +178,10 @@ test('[ReportScreen] should render ReportScreen with composer interactions', ()
         await waitFor(triggerTransitionEnd);
 
         // Query for the composer
-        const composer = await screen.findByTestId('composer');
-
-        // Type in the composer
-        fireEvent.changeText(composer, 'Test message');
-
-        const hintSendButtonText = Localize.translateLocal('common.send');
-
-        // Query for the send button
-        const sendButton = await screen.findByLabelText(hintSendButtonText);
-
-        // Click on the send button
-        fireEvent.press(sendButton);
-
-        const hintHeaderText = Localize.translateLocal('common.back');
-
-        // Query for the header
-        await screen.findByLabelText(hintHeaderText);
-    };
-
-    const navigation = {addListener};
-
-    return waitForBatchedUpdates()
-        .then(() => {
-            const reportCollectionDataSet: ReportCollectionDataSet = {
-                [`${ONYXKEYS.COLLECTION.REPORT}${mockRoute.params.reportID}`]: report,
-            };
-
-            const reportActionsCollectionDataSet: ReportActionsCollectionDataSet = {
-                [`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${mockRoute.params.reportID}`]: reportActions,
-            };
-
-            return Onyx.multiSet({
-                [ONYXKEYS.IS_SIDEBAR_LOADED]: true,
-                [ONYXKEYS.PERSONAL_DETAILS_LIST]: personalDetails,
-                [ONYXKEYS.BETAS]: [CONST.BETAS.DEFAULT_ROOMS],
-                [`${ONYXKEYS.COLLECTION.POLICY}`]: policies,
-                [ONYXKEYS.SHOULD_SHOW_COMPOSE_INPUT]: true,
-                ...reportCollectionDataSet,
-                ...reportActionsCollectionDataSet,
-            });
-        })
-        .then(() =>
-            measurePerformance(
-                <ReportScreenWrapper
-                    // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
-                    navigation={navigation}
-                    // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
-                    route={mockRoute}
-                />,
-                {scenario},
-            ),
-        );
-});
-
-test.skip('[ReportScreen] should press of the report item', () => {
-    const {triggerTransitionEnd, addListener} = TestHelper.createAddListenerMock();
-    const scenario = async () => {
-        /**
-         * First make sure ReportScreen is mounted, so that we can trigger
-         * the transitionEnd event manually.
-         *
-         * If we don't do that, then the transitionEnd event will be triggered
-         * before the ReportScreen is mounted, and the test will fail.
-         */
-        await screen.findByTestId('ReportScreen');
-
-        await waitFor(triggerTransitionEnd);
+        await screen.findByTestId('composer');
```

Comment:
so the only thing we'll test here is how long/much it takes for the screen to render with a composer, correct? What are the sample results you were getting for this scenario?

Reply:
Is typing anything here still dependant on a bunch of props?

Reply:
Yes. I think this is enough to see when regression occurs.  All interactions have already been tested here:  https://github.com/Expensify/App/blob/main/tests/perf-test/ReportActionCompose.perf-test.tsx. 

<img width="837" alt="Screenshot 2024-04-18 at 12 33 43" src="https://github.com/Expensify/App/assets/19835085/c9848efd-f778-4dfd-955b-c40e8bbaaf4c">